src/Services/Manager.vala

```diff
@@ -98,14 +99,6 @@ public class Bluetooth.Services.ObjectManager : Object {
                                 adapter.stop_discovery.begin ();
                             }
                         }
-                        
-                    }
-
-                    var discoverable = changed.lookup_value("Discoverable", new VariantType("b"));
```

Comment:
Is there a reason this was removed? This looks like the mechanism that ensures that we remain discoverable while the plug is open

Reply:
As far as I could see it does nothing.  It appears to be saying if the value of adapter.discoverable changes then ensure that adapter.discoverable is what it was signalled to have changed to anyway??